R/expect-condition.R

```diff
@@ -106,6 +110,25 @@ expect_condition <- function(object,
   invisible(act$val %||% act$cap)
 }
 
+capture_first_condition <- function(expr) {
```

Comment:
Should be in capture-conditions.R?

Reply:
I'll kept it here for a while since it makes it easier to check the code coverage.